pyccel/commands/pyccel_wrap.py

```diff
@@ -193,5 +193,3 @@ def pyccel_wrap_command() -> None:
                        conda_warnings  = args.conda_warnings)
     except PyccelError:
         sys.exit(1)
-    finally:
-        os.chdir(base_dirpath)
```

Comment:
I think that `base_dirpath` is not used at all after this change

Reply:
Fixed : [f254a58](https://github.com/pyccel/pyccel/pull/2476/commits/f254a588e283509441dff555089c070ecafe21e9)